backend/src/main/java/de/neuefische/capstone/backend/imageupload/ImageUploadRepo.java

```diff
@@ -6,5 +6,4 @@
 
 @Repository
 public interface ImageUploadRepo extends MongoRepository<Image, String> {
-
 }
```

Comment:
I still don't know if it actually makes a difference, but you are missing an empty line here 🙈